Update subscriptions docs with 6.x changes (#192)
neo4j/docs-graphql: modules/ROOT/pages/subscriptions/customize-cdc.adoc

[[customize-cdc]]

> = Customize CDC Subscriptions

Reviewer: ```suggestion
= Customize CDC subscriptions
```

:page-aliases: subscriptions/plugins/index.adoc, subscriptions/plugins/amqp.adoc, subscriptions/plugins/single-instance.adoc

> :description: This page describes how  to customize the behaviour of subscriptions.

Reviewer: ```suggestion
:description: This page describes how to customize the behavior of subscriptions.
```

> 

Reviewer: we should mention what CDC stands for. we don't have an introductory section yet - maybe that would be a good excuse to create one :)

== Neo4jGraphQLSubscriptionsCDCEngine

> By default, the GraphQL library will use the same driver passed to `Neo4jGraphQL` to poll for events every second. This behaviour can be changed by creating a custom instance of `Neo4jGraphQLSubscriptionsCDCEngine`.

Reviewer: ```suggestion
By default, the GraphQL library uses the same driver passed to `Neo4jGraphQL` to poll for events every second.
This behavior can be changed by creating a custom instance of `Neo4jGraphQLSubscriptionsCDCEngine`.
```

The following options can be passed to the constructor:

* `driver`: The driver to be used for CDC queries.
* `pollTime`: The interval, in milliseconds, between queries to CDC. 
Defaults to 1000ms.

> Note that poll time is the period between one request finishing and the next one starting. 

Reviewer: ```suggestion
Note that poll time is the period between a finished request and the start of the next. 
```

> The actual time it takes for CDC events to trigger the subscription also depend on your network.

Reviewer 2: ```suggestion
The actual time it takes for CDC events to trigger the subscription also depends on your network.
```

* `queryConfig`: An object with the driver query options to be passed to CDC requests. 
Use the `db` field to define the target database for CDC. 

> For example:

Reviewer 2: It might be nice to include queryConfig in the example too

[[customize-cdc]]
= Customize CDC subscriptions
:page-aliases: subscriptions/plugins/index.adoc, subscriptions/plugins/amqp.adoc, subscriptions/plugins/single-instance.adoc
:description: This page describes how to customize the behavior of subscriptions.

> GraphQL subscriptions to Neo4j use [Change Data Capture](https://neo4j.com/docs/cdc/current/). It's behaviour can be configured by passing an instance of `Neo4jGraphQLSubscriptionsCDCEngine`

Reviewer: ```suggestion
GraphQL subscriptions to Neo4j use [Change Data Capture](https://neo4j.com/docs/cdc/current/) (CDC).
It's behavior can be configured by passing an instance of `Neo4jGraphQLSubscriptionsCDCEngine`.
```

[[customize-cdc]]
= Customize CDC subscriptions
:page-aliases: subscriptions/plugins/index.adoc, subscriptions/plugins/amqp.adoc, subscriptions/plugins/single-instance.adoc
:description: This page describes how to customize the behavior of subscriptions.

GraphQL subscriptions to Neo4j use [Change Data Capture](https://neo4j.com/docs/cdc/current/) (CDC).

> It's behavior can be configured by passing an instance of `Neo4jGraphQLSubscriptionsCDCEngine`.

Reviewer: ```suggestion
Its behavior can be configured by passing an instance of `Neo4jGraphQLSubscriptionsCDCEngine`.
```
